content/tracing/advanced_usage/_index.md

```diff
@@ -1408,18 +1408,34 @@ The purpose of this section is to explain how the correlation between traces and
 {{< tabs >}}
 {{% tab "Java" %}}
 
-Leverage the MDC Frameworks ([Map Diagnostic Context][1]) to automatically add the trace and span ids into your logs. Datadog MDC keys may be injected automatically with a tracer integration, or manually injected with Datadog APIs.
+Leverage the MDC Frameworks ([Map Diagnostic Context][1]) to automatically correlate trace and span ids into your logs. Datadog MDC keys may be injected automatically with a tracer integration, or manually injected with Datadog APIs.
 
-**How to enable Log Injection***
+**Enable Log Injection for JSON Formatted Logs**
 
 1. Enable injection in the [Java Tracer's configuration][3]. Note: Currently only slf4j is supported for MDC autoinjection.
-2. Alternatively, Use Datadog `CorrelationIdentifier#getTraceId()`, and `CorrelationIdentifier#getSpanId()` APIs to inject identifiers at the beginning and end of each span to log (see examples below).
-3. Configure MDC to use the injected Keys:
-  * `dd.trace_id` Active trace id during the log statement (or `0` if no trace)
-  * `dd.span_id` Active trace id during the log statement (or `0` if no trace)
-4. If logs are already JSON formatted, there should be nothing left to do. [See our Java logging documentation][2] to add those two identifiers in raw logs or to learn how to log in JSON.
 
-**Manual Injection log4j2 example**:
+**Enable Log Injection for Raw Formatted Logs**
+
+1. Enable injection in the [Java Tracer's configuration][3]. Note: Currently only slf4j is supported for MDC autoinjection.
+2. Update your formatter to include `dd.trace_id` and `dd.span_id` in your logs
+
+```xml
+<Pattern>"%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %X{dd.trace_id} %X{dd.span_id} - %m%n"</Pattern>
```

Comment:
this is a duplication from https://docs.datadoghq.com/logs/log_collection/java/?tab=slf4j#configure-your-logger but I prefer to have everything in one place rather than pointing people to different parts of the docs.

I think in the future we may link both sides so that we pull the content from logs section to avoid any kind of duplication. But for now, I think it's good enough.

Reply:
I think this is a good call.